ansible/playbooks/ptf_installation.yaml

```diff
@@ -0,0 +1,60 @@
+---
+- name: PTF Installation
+  hosts: hana
+  remote_user: cloudadmin
+  become: true
+  become_user: root
+  vars:
+    ptf_url: "your_ptf_url"
+    ptf_user: "your_username_here"
+    ptf_password: "your_pass_here"
+    ptf_dir: "/tmp/ptf_dir"
+  tasks:
+    - name: Create directory for PTF installation
+      file:
+        path: "{{ ptf_dir }}"
+        state: directory
+        mode: '0755'
+
+    - name: Download PTF files recursively with wget
+      command: "wget --no-directories --recursive --reject 'index.html*' --user={{ ptf_user }} --password={{ ptf_password }} --no-parent {{ ptf_url }}"
+      args:
+        chdir: "{{ ptf_dir }}"
+
+    - name: List downloaded files
+      command: "ls -la"
+      args:
+        chdir: "{{ ptf_dir }}"
+      register: download_list
```

Comment:
is i t needed to register and then use a `debug:` task? Is not enough to have the `command` task?